pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/service/SomaticResultUploadServiceTest.java

```diff
@@ -3,22 +3,83 @@
 import static org.junit.Assert.assertNotNull;
 
 import com.typesafe.config.Config;
+import org.broadinstitute.ddp.db.TransactionWrapper;
 import org.broadinstitute.ddp.util.ConfigManager;
+import org.broadinstitute.dsm.DbAndElasticBaseTest;
+import org.broadinstitute.dsm.db.SomaticResultUpload;
+import org.broadinstitute.dsm.db.dao.ddp.instance.DDPInstanceDao;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDto;
+import org.broadinstitute.dsm.model.somatic.result.SomaticResultMetaData;
+import org.broadinstitute.dsm.util.DdpInstanceGroupTestUtil;
+import org.broadinstitute.dsm.util.ElasticTestUtil;
+import org.broadinstitute.dsm.util.TestParticipantUtil;
+import org.junit.AfterClass;
+import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-public class SomaticResultUploadServiceTest {
+import java.time.Duration;
+import java.time.Instant;
 
-    static SomaticResultUploadService somaticResultUploadSerivce;
+public class SomaticResultUploadServiceTest extends DbAndElasticBaseTest {
+
+    private static final DDPInstanceDao ddpInstanceDao = new DDPInstanceDao();
+    private static final String instanceName = "somatic-upload-test";
+    private static String esIndex;
+    private static DDPInstanceDto ddpInstanceDto;
+    private static ParticipantDto testParticipant = null;
 
     @BeforeClass
-    public static void setup() {
+    public static void setup() throws Exception {
+        esIndex = ElasticTestUtil.createIndex(instanceName, "elastic/lmsMappings.json", null);
+        ddpInstanceDto = DdpInstanceGroupTestUtil.createTestDdpInstance(instanceName, esIndex, instanceName);
         Config loadedConfig = ConfigManager.getInstance().getConfig();
         somaticResultUploadSerivce = SomaticResultUploadService.fromConfig(loadedConfig);
     }
 
+    @AfterClass
+    public static void tearDown() {
+        try {
+            Assert.assertNotNull(testParticipant);
```

Comment:
I suggest deleting `testParticipant` in an after method class, since it is setup by a particular test.
(This will matter when another test is added.)

Reply:
Oooh good catch!

Reply:
Will fix this by creating the test participant once in the beforeClass.  There are already a few tests in this class, but only one makes use of the generated test participant, so when additional tests are added, they can (hopefully) use the same participant.